src/main/java/org/terasology/structureTemplates/internal/systems/AddItemsToChestSystem.java

```diff
@@ -34,6 +34,7 @@
 
 /**
  * System to power the {@link AddItemsToChestComponent}
+ * Adds items to chests specified by the AddItemsToChestComponent.
```

Comment:
The {@link ClassName} syntax like in the line above can be used to create a link to the class.

I think the above sentence is enough, as you can read up what the component does when you click on it.